src/components/widgets-settings/forecast-payroll/forecast-payroll.tmpl.html

```diff
@@ -3,7 +3,7 @@ <h5 translate>impac.widget.settings.forecast_payroll.title</h5>
 
   <div class="widget-lines-container">
     <div class="widget-line">
-      Aggregate
+      {{'impac.widget.account_cash_projection.forecast_payroll.aggregate' | translate}}
```

Comment:
@agranado2k when not interpolating a translate alongside other text, prefer this syntax
```html
<div class="widget-line" translate>
  impac.widget.account_cash_projection.forecast_payroll.aggregate
</div>
```